code/src/index.css

```diff
@@ -1,13 +1,296 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+* {
+  box-sizing: border-box;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+button {
+  border: none;
+  outline: none;
+  background: none;
+}
+
+/*---- FONT IMPORTS -----------   */ 
+
+@font-face {
+  font-family: 'Circular Book';
+  src: url('./fonts/c-book.eot'), /* IE9+ Compat Modes */
+       url('./fonts/c-book.woff') format('woff'), /* Modern browsers */
+       url('./fonts/c-book.ttf')  format('truetype'); /* Safari, Android, iOS */
+}
+
+@font-face {
+  font-family: 'Circular Bold';
+  src: url('./fonts/c-bold.eot'); /* IE9+ Compat Modes */
+  src: url('./fonts/c-bold.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('./fonts/c-bold.woff') format('woff'), /* Modern browsers */
+       url('./fonts/c-bold.ttf')  format('truetype'); /* Safari, Android, iOS */
+}
+
+@font-face {
+  font-family: 'Circular Black';
+  src: url('./fonts/c-black.eot'); /* IE9+ Compat Modes */
+  src: url('./fonts/c-black.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('./fonts/c-black.woff') format('woff'), /* Modern browsers */
+       url('./fonts/c-black.ttf')  format('truetype'); /* Safari, Android, iOS */
+}
+
+
+/* ------- STYLING --------- */ 
+
+body {
+  padding: 16px;
+  color: #fff;
+  background-color: #181818;
+  margin: 0;
+  font-family: 'Circular Book',sans-serif;
+}
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
+  margin-top: 48px;
+  border-bottom: 1px solid #282828;
+  margin-bottom: 15px;
+  padding-bottom: 6px;
+  font-size: 18px;
+  letter-spacing: -0.022em;
+  font-family: "Circular Black";
+}
+
+.content {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.albums {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.album {
+  /*min-width: 200px; */
```

Comment:
It's considered as good practice to remove all commented code before pushing it to Github - let's do that